src/components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -81,12 +79,14 @@ const ConsultationDoctorNotes = (props: ConsultationDoctorNotesProps) => {
       },
     });
 
-    if (res?.status === 201) {
-      Notification.Success({ msg: "Note added successfully" });
-      setState({ ...state, cPage: 1 });
+    if (res?.status === 201 && data) {
       setNoteField("");
-      setReload(true);
+      setState((prevState) => ({
+        ...prevState,
+        notes: [data, ...prevState.notes],
+      }));
```

Comment:
you can use tanstack's useMutation to add the note and perform perform query invalidations for the query that fetches the notes